src/models/OAuthModel.php

```diff
@@ -125,20 +125,25 @@ public function createAccessTokenFromPassword($clientId, $username, $password)
      */
     protected function getUserId($username, $password)
```

Comment:
Working on a controller test for this, however for some weird reason the PDO mock isn't being passed from the TokenController to the OAuthModel, looking into it to see what I can do